torchvision/models/quantization/shufflenetv2.py

```diff
@@ -22,35 +26,35 @@
 
 
 class QuantizableInvertedResidual(shufflenetv2.InvertedResidual):
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
         super(QuantizableInvertedResidual, self).__init__(*args, **kwargs)
         self.cat = nn.quantized.FloatFunctional()
 
-    def forward(self, x):
+    def forward(self, x: Tensor) -> Tensor:
         if self.stride == 1:
             x1, x2 = x.chunk(2, dim=1)
-            out = self.cat.cat((x1, self.branch2(x2)), dim=1)
+            out = self.cat.cat((x1, self.branch2(x2)), dim=1)  # type: ignore
```

Comment:
I'm not sure but `cat.cat` does not work well with mypy.

Reply:
mind specifying the error type? (and perhaps posting here the error so we check if there is a cleaner way around)